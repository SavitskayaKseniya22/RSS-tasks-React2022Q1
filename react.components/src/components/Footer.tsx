import React from 'react';
import './footer.css';
import logo from '../assets/svg/rs-school-js.svg';

export class Footer extends React.Component {
  constructor(props: string) {
    super(props);
  }
  render() {
    return (
      <footer className="Footer">
        <div>
          <a href="https://rs.school/js/" target="_blank" rel="noreferrer">
            <img src={logo} alt="logo" width="100" />
          </a>
        </div>
        <div>
          <a href="https://github.com/SavitskayaKseniya22" target="_blank" rel="noreferrer">
            made by Kseniya Savitskaya
          </a>
        </div>
        <div>Â© 2022</div>
      </footer>
    );
  }
}
