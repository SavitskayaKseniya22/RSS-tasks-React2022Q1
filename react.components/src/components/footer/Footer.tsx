import './footer.css';
import logo from '../../assets/svg/rs-school-js.svg';

export function Footer() {
  return (
    <footer className="footer" data-testid="footer">
      <div>
        <a href="https://rs.school/js/" target="_blank" rel="noreferrer">
          <img src={logo} alt="logo" width="100" />
        </a>
      </div>
      <div>
        <a href="https://github.com/SavitskayaKseniya22" target="_blank" rel="noreferrer">
          made by Kseniya Savitskaya
        </a>
      </div>
      <div>Â© 2022</div>
    </footer>
  );
}
