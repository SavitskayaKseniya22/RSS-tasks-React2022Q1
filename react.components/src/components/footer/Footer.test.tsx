import { render, screen } from '@testing-library/react';
import { Footer } from './Footer';

test('check footer appearance', () => {
  render(<Footer />);
  const footer = screen.getByTestId('footer');
  expect(footer).toMatchSnapshot();
});
