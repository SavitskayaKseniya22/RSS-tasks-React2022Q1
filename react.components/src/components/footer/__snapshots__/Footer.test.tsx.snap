// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check footer appearance 1`] = `
<footer
  class="footer"
  data-testid="footer"
>
  <div>
    <a
      href="https://rs.school/js/"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="logo"
        src="rs-school-js.svg"
        width="100"
      />
    </a>
  </div>
  <div>
    <a
      href="https://github.com/SavitskayaKseniya22"
      rel="noreferrer"
      target="_blank"
    >
      made by Kseniya Savitskaya
    </a>
  </div>
  <div>
    Â© 2022
  </div>
</footer>
`;
